package.json

```diff
@@ -1,10 +1,11 @@
 {
   "name": "clarity-js",
-  "version": "0.1.44",
+  "version": "0.2.0",
```

Comment:
Why the bump to 0.2?

Reply:
Breaking changes in the exported module structure. This would prevent somebody who's using v0.1.x from accidentally installing this version by just running "npm install"

Reply:
Sorry if I missed a conversation between you and Swaathee - why do the dashboard and analytics services needs all of these changes?

---
In reply to: [215666151](https://github.com/Microsoft/clarity-js/pull/104#discussion_r215666151) [](ancestors = 215666151)

Reply:
There was no such conversation - this is my initiative. Tslint in dashboard requires typed variables in functions. Therefore, moving visualization event parsers to dashboard needs to have access to most Clarity interfaces. I don't want to add hacky "node_modules/..... /clarity.d.ts" or /// <reference path=".." /> like we have now, hence the change for proper typing setup on the package.